packages/core/content-type-builder/admin/src/components/FormModalEndActions/index.js

```diff
@@ -383,6 +385,42 @@ const FormModalEndActions = ({
     );
   }
 
+  if (isCustomFieldModal) {
```

Comment:
Woow, this component is hell 🔥 . Do you think this is a good chance to split it up a bit? 👀

Reply:
I don't mind cleaning this component up a bit, but what did you have in mind, moving each button group to its own component?

Reply:
Yes, I think that would be an improvement in terms of readability. Wdyt? Could also be a task after this PR to keep the scope a bit smaller ...

Reply:
In terms of timing it might be better to leave this as is until the alpha and make the improvement before the beta